### DISCUSSION TITLE
Saving only time-series file after processing fMRI

### DISCUSSION THREAD

**Discussion by shivamkrsharma44** [FCP-INDI/C-PAC#1602] [Q&A]
If I just want the time-series data of a subject say for CC200 atlas, how do I do that ?
I don't want to save the processed fMRI files in the output directory. How do I turn that off in the 'default_pipeline.yml' file?

**Comment by sgiavasis** [comment]
Hello @shivamkrsharma44,

Currently, the preprocessed BOLD data is one of the default outputs of CPAC and cannot be disabled. This isn't a bad idea, I'm going to create an issue for it. We simply haven't had this request before and it's usually the main output users are looking for.

Is the main concern the amount of space the timeseries data takes up?

Steve

**Reply by sgiavasis** [reply]
Ref: https://github.com/FCP-INDI/C-PAC/issues/1605

**Reply by shivamkrsharma44** [reply]
Thank you @sgiavasis . 

Precisely,  space is one of the main concerns when dealing with a large dataset. It would be great if there is an option to save just the time-series files for different atlases.
